.github/workflows/release.yml

```
@@ -0,0 +1,90 @@
+      - name: 🧪 Run tests
+        run: |
+          npm run test:run
+          npm run lint
+
+      - name: 🔨 Build
+        run: npm run build
+
+      - name: 📤 Create Sentry release
+        env:
+          SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:5:55: Double quote to prevent globbing and word splitting

(shellcheck)